### ISSUE TITLE
FCM Sdk 4.2.3 crashes in runtime with AndroidX migrated project

### ISSUE THREAD

**Issue by SemenovAlexander** [Leanplum/Leanplum-Android-SDK#324]
Hi guys. I've updated leanplum SDK to 4.2.3, which forced me to migrate the project to AndroidX. After that migration app crashes trying to access any of SDK classes.

## Expected Behavior
SDK doesn't crash 

## Actual Behavior
```
java.lang.VerifyError: Rejecting class com.osemenov.myapplication.FCMJavaService that attempts to sub-type erroneous class com.leanplum.LeanplumPushFirebaseMessagingService (declaration of 'com.osemenov.myapplication.FCMJavaService' appears in /data/app/com.osemenov.myapplication-erh9XnEEZy84ccbuEww1oQ==/base.apk)
        at com.osemenov.myapplication.FCMJavaService.test(FCMJavaService.java:7)
        at com.osemenov.myapplication.MainActivity.onCreate(MainActivity.java:11)
```

## Steps to Reproduce the Problem

  1. Migrate project to AndroidX
  2. Add leanplum-fcm SDK 4.2.3
  3. Access any SDK class at runtime
See demo project - just compile it and launch MainActivity
https://github.com/SemenovAlexander/LeanplumAndroidXBug

## Specifications

  - Version: 4.2.3
  - Platform: Android 9.0

**Comment by SemenovAlexander** [comment]
Guys, any updates here? #333 is basically the same issue

**Comment by lolsheeplol** [comment]
@SemenovAlexander Which version are you having problems with?  I see 4.3.2 in the title and then 4.2.3 in the body.

**Comment by SemenovAlexander** [comment]
@lolsheeplol it is 4.2.3, sorry about that

**Comment by e7mac** [comment]
This is a known issue in version 4.3.0. This issue was resolved in 4.3.1
We had some issues with deploying 4.2.3 and it's possible that you might have a dirty cache.
Please could you update to 4.3.1 or above and see if that resolves the issue?
